File: hbe-tests/projects/TEST-07-realistic/app/src/main/java/com/hbe/test07/MainActivity.java
package com.hbe.test07;
import android.os.Bundle;
import android.widget.TextView;
import androidx.appcompat.app.AppCompatActivity;
public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        TextView tv = new TextView(this);
        tv.setText("TEST-07-realistic");
        setContentView(tv);
    }
}

File: hbe-tests/projects/TEST-08-version-catalog/app/src/main/kotlin/dev/hbe/test08/MainActivity.kt
package dev.hbe.test08

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val tv = TextView(this)
        tv.text = "TEST-08 Version Catalog Works!"
        setContentView(tv)
    }
}
